contributors.jenkins.io.tf

```diff
@@ -26,3 +30,8 @@ resource "azurerm_storage_account" "contributorsjenkinsio" {
   tags = local.default_tags
 }
 
+resource "azurerm_storage_share" "contributors_jenkins_io" {
+  name                 = "contributors-jenkins-io"
+  storage_account_name = azurerm_storage_account.contributors_jenkins_io.name
+  quota                = 5
```

Comment:
Could you explain ("posteriori") the reasoning behind this value in the issue please?

Reply:
I've chose 5Gio as even javadoc (another webiste) with a lot of content is below 4Gio.